src/data_standardization/tests/test_standardize_point_data.py

```diff
@@ -23,18 +23,17 @@ def create_test_csv(test_list, tmpdir, filename):
 
 
 def test_read_file_info_default(tmpdir):
-
     test = [{
         'Ticket Issue Date': '09/02/2014',
         'Issue Time': '9:10 AM',
         'Location': "1100 CAMBRIDGE ST\nCambridge, MA\n" +
-        "(42.37304187600046, -71.09569369699966)",
+                    "(42.37304187600046, -71.09569369699966)",
```

Comment:
what happened with the indenting here?

Reply:
Uhh I dunno.  PyCharm maybe did something to it? I'll fix throughout.